packages/forms/src/index.tsx

```diff
@@ -315,7 +332,7 @@ export const CheckboxField = forwardRef<
   const type = 'checkbox'
 
   React.useEffect(() => {
-    if (process.env.NODE_ENV !== 'production' && props.dataType !== undefined) {
+    if (props.dataType !== undefined) {
```

Comment:
How come you removed the NODE_ENV check entirely here?

Reply:
Not sure ... my error.  Thanks for catching!